src/components/Iframe.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
+import Requests from "./Requests";
+import "../styles/iframe.css";
+
+export default class Iframe extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      game: {}
+    };
+  }
+
+  async componentDidMount() {
+    const id = this.props.match.params.id;
+    this.setState({ game: await Requests.get("/games/" + id) });
+    console.log(this.state.game);
```

Comment:
You can remove the console.log also use string literal to concatenate the game id